aws-android-sdk-iot/src/main/java/com/amazonaws/mobileconnectors/iot/AWSIotMqttManager.java

```diff
@@ -972,19 +1017,58 @@ public void connect(
             final String customAuthorizer,
             final AWSIotMqttClientStatusCallback statusCallback) {
 
-        if (tokenKeyName == null || token == null || tokenSignature == null) {
+        if (tokenKeyName == null || token == null) {
```

Comment:
The stipulation in the documentation is that we will no longer complain about token signature stuff _if_ the user is using a custom endpoint. Do we still have a safeguard here in the case where the user is _not_ using custom auth features?

Reply:
Good point. Token-signature is optional for all types of endpoints not just custom ones, if token-signing is disable in the domain configuration. Old, existing IoT endpoint however do not support that as yet. Will make a note about it in the comment as discussed.